### ISSUE TITLE
Image branding

### ISSUE THREAD

**Issue by sergiomeneses** [houseme/qrcode-rs#9]
Hey, just wanna know if there is wayt to brand the QR code generated, like colors and logo.

**Comment by houseme** [comment]
It is not supported at present. You are welcome to contribute. @sergiomeneses

**Comment by houseme** [comment]
Yes, the renderer supports custom colors via `dark_color()` and `light_color()`:

```rust
let image = QrCode::new(b"hello").unwrap()
    .render::<Rgb<u8>>()
    .dark_color(Rgb([0, 0, 128]))
    .light_color(Rgb([224, 224, 224]))
    .build();
```

For logo overlay, the `image` crate provides methods to composite/overlay images on top of the generated QR code. This is best done at the application level since logo placement and sizing depend on your specific design requirements.
